plugins/module_utils/api.py

```diff
@@ -44,11 +44,12 @@ def api_argument_spec():
         validate_certs=dict(type='bool', default=True),
         validate_cert_hostname=dict(type='bool', default=False),
         ca_path=dict(type='path'),
-        encoding=dict(type='str', default='ASCII')
+        encoding=dict(type='str', default='ASCII'),
+        timeout=dict(type='int', default=10)
```

Comment:
Then the next diff will be cleaner :)
```suggestion
        timeout=dict(type='int', default=10),
```

Reply:
fixed